cmd/proxy/assets/insecure.html

```diff
@@ -36,7 +36,56 @@
     </script>
   </head>
   <body>
-    <div class="container">
+
+      <div class="flex-column flex1">
+        <div class="tlsContent--wrapper">
+          <div>
+            <p class="tls-header">Bypass browser TLS warning</p>
+            <p class="tls-header-sub">We use a self-signed SSL/TLS Certificate to secure the communication between your local machine and the Admin Console during setup. You'll see a warning about this in your browser, but you can be confident that this is secure.</p>
+          </div>
+          <div class="flex1 flex">
+            <div class="flex-column flex1 left-block">
+              <div>
+                <p class="tls-section-header"><script>document.write(browserPlainText)</script></p>
+                <p class="tls-section-sub"><script>document.write(browserBodyText)</script></p>
+              </div>
+              <div class="CodeSnippet">
+                <div class="CodeSnippet-content">
+                  <p>Verifying the certificate's authenticity</p>
+                  <pre>
+                    <code>$ echo | openssl s_client -servername local -connect ' + opensslLink + ' 2>/dev/null | openssl x509 -noout -fingerprint</code>
+                  </pre>
+                </div>
+              </div>
+              <div class="CodeSnippet">
+                <div class="CodeSnippet-content">
+                  <p>SHA Fingerprint</p>
+                  <pre>
+                    <code>{{ .fingerprintSHA1 }}</code>
+                  </pre>
+                </div>
+              </div>
+              <div class="button-wrapper flex alignItems--center">
+                <button type="submit" class="btn primary" onclick="location.href=tlsManagementLink;">Continue to Setup</button>
+                <p class="text-muted">
+                  <script>document.write(
+                    'or visit <a class="replicated-link" href="' + httpsLink + '">' + httpsLink + '</a> to proceed to application'
+                  )</script>
+                </p>
+              </div>
+            </div>
+            <div class="flex-column flex1 right-block alignItems--center justifyContent--center">
+              <script>document.write(
+                '<img class="insecure-image" src="/assets/images/' + browserPlainText.toLowerCase().replace(" ", "") + '-insecure.png" />'
```

Comment:
.replace() will not replace all occurrences? is that intentional? if not, you can do something like this `.replace(/ /g, '');

Reply:
I think it's ok for this yes

Reply:
cool